db/migrate/20210621103509_add_bannered_until.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddBanneredUntil < ActiveRecord::Migration[6.1]
+  def change
+    add_column :topics, :bannered_until, :datetime, null: true
```

Comment:
I think we need a partial index here. Otherwise, the ensure consistency query is going to be very slow since we have to query against the entire topics table.

Reply:
We don't have a partial index on `pinned_until` either